components/OutivityDetail/index.js

```diff
@@ -42,4 +57,21 @@ const StyledCountryName = styled.span`
 const StyledOutivityImage = styled(Image)`
   overflow: clip;
   overflow-clip-margin: content-box;
+  height: auto;
```

Comment:
height="auto" could mess with the pages layout until its fully loaded. consider using the layout="fixed"-prop on the Image itself.

Reply:
Thank you for the tip 😄